src/components/blend-model.js

```diff
@@ -0,0 +1,40 @@
+var registerComponent = require('../core/component').registerComponent;

```

Comment:
Should this be a standard component? It could fit alongside the hand-controls example.